README.md

```diff
@@ -1,5 +1,10 @@
 # ItsHackathon
 
+## Stack
+
+- coderabbit
+- codium pr analyzer
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of "coderabbit" and "codium pr analyzer" to the stack is mentioned, but there's no explanation or context provided about what these tools are used for. It would be helpful to include a brief description or link to documentation for each tool in the stack.
```diff
  ## Stack
  
- - coderabbit
- - codium pr analyzer
+ - [coderabbit](<link_to_coderabbit_documentation>) - <brief_description_of_coderabbit>
+ - [codium pr analyzer](<link_to_codium_pr_analyzer_documentation>) - <brief_description_of_codium_pr_analyzer>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->